### DISCUSSION TITLE
cant add files to the repository

### DISCUSSION THREAD

**Discussion by Priyadharshini0315** [community/community#35876] [Other Feature Feedback, Questions, & Ideas]
![WhatsApp Image 2022-10-02 at 1 26 45 PM](https://user-images.githubusercontent.com/114298710/195281179-fa121714-fa08-4529-8628-8ceb68b9941c.jpeg)

it shows the repository is empty how can i add files ,pls help me out

**Comment by ihnatenkoo** [comment]
Hello, see here:
**[Adding a file to a repository](https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository)**

**Comment by MitkoVtori** [comment]
I see your files. They appear idk why you don't see them.
